### ISSUE TITLE
Qwant engine displays its results in French or German all the time

### ISSUE THREAD

**Issue by Immortality-IMT** [searxng/searxng#3726]
2024.6.23+9a9ca307f+dirty 
Installed using install script.

Issue:
The qwant search engine displays all its results in French or German all the time and is enabled by default.

Solution:
The engine disabled by default "disabled: true" and appear in the Engine preferences as Qwant (FR) next to navar(KO)

**Comment by return42** [comment]
> 2024.6.23+9a9ca307f+dirty

You have a (old and) modified instance, nearly all you wrote in your ticket is wrong and confusing / we do not have "Qwant (FR)" .. not sure what config you use .. and the qwant engine works / except the CAPTCHA problem.

**Your experience comes from an outdated copy of SearXNG that you are tinkering with**

You have already made several misleading posts here that will only confuse any newcomer and lead them in the wrong direction.  I constantly have to clean up after you and make things clear.

Stop this nonsense, check your statements in an actual clone of this project that is not modified.

**To be clear in the future I will delete such posts & issues from yours without any further  comment.**

**Comment by Immortality-IMT** [comment]
Version: 2024.8.9+f1c05e7c1 —
![Screenshot from 2024-08-11 17-50-35](https://github.com/user-attachments/assets/7053e829-f22e-45e3-b799-f37f60df7978)
I just updated the version and it has not changed. The qwant engine only returns foreign language results.
![Screenshot from 2024-08-11 17-53-23](https://github.com/user-attachments/assets/a3994001-94f6-45ce-beb9-06b1c8973844)
The engine is not represented with (FR) like the others are.
TBH: I think the search engine outputs gibberish. I don't know what language it is.

**Comment by return42** [comment]
TL;DR; .. qwant engine has no issues except the ongoing CAPTCHA problem / https://github.com/searxng/searxng/issues/2454

> TBH: I think the search engine outputs gibberish. I don't know what language it is.

What language do you selected? .. try search query`!qwant :en bmw` and you will get english results:

![image](https://github.com/user-attachments/assets/ca2aff97-0e26-47fb-86f7-014b25791b6b)

or german results on `:de`:

![image](https://github.com/user-attachments/assets/38aebe79-6ec7-4e94-a148-8475c1429e93)

or Chinese results:

![image](https://github.com/user-attachments/assets/d31bdf3d-84ea-45ad-b577-59d504aceffe)


The engine work within all languages supported by the origin qwant, will return all the results the origin qwant will return on the query:

https://github.com/searxng/searxng/blob/f1c05e7c16e59a8d5d98640bc057958ea268cddf/searx/data/engine_traits.json#L5939-L5985

**Comment by Immortality-IMT** [comment]
All the options are default, fresh installation.

Auto-detect (en-US)

Still outputs gibberish

![Screenshot from 2024-08-11 18-57-59](https://github.com/user-attachments/assets/516d5fa8-48d0-4430-a337-df91bc86dce5)

Visiting the website returns
Thanks for your visit
Unfortunately we are not yet available in your country.

**Comment by return42** [comment]
> Unfortunately we are not yet available in your country.

where is your (server) IP located?

**Comment by Immortality-IMT** [comment]
Vietnam, but I also used a VPN, USA based and still gibberish.
Using the VPN and going to a website, it tries to auto guess the region and auto-translate but gets it wrong.
It has region settings on the websites.

**Comment by return42** [comment]
> used a VPN, USA based and still gibberish.

don't believe .. qwant is available in USA .. test origin qwant and then SearXNG from your VPN IP in USA and both will work.

**Comment by Immortality-IMT** [comment]
Returns gibberish with and without VPN. qwant.com changed the locale from US to British, and now returns gibberish.

![Screenshot from 2024-08-11 19-32-31](https://github.com/user-attachments/assets/a03f89ca-011c-4d8a-8349-26e81b5feab9)

**Comment by return42** [comment]
Sorry, I don't know whats going wrong and why you get gibberish from origin qwant .. but this also shows that the SearXNG works perfectly ... it passes us what it gets from qwant origin.
